blog/en/blog/2022/10/31/apache-apisix-v3-preview.md

```diff
@@ -40,93 +36,88 @@ But just solving these problems is not enough. As a middleware with requests fro
 
 In addition to business-level requirements, API gateways also have requirements for customization and integration. So how to make custom development easier for developers when using APISIX is another significant pain point that APISIX solves, lowering the threshold for developers to code.
 
-In APISIX, plugins are developed mainly through Lua, and [LuaJIT](https://apisix.apache.org/blog/2021/08/25/why-apache-apisix-chose-nginx-and-lua/) (a Just-In-Time Compiler for Lua) is used to ensure that the compiled code performance is good enough. However, as a relatively minor language, the learning cost of Lua is high for most backend engineers. To solve this problem, APISIX provides two solutions.
-
-## Support More Programming Languages
-
-The first solution is to support more widely-used programming languages ​​(such as [Java](https://github.com/apache/apisix-java-plugin-runner), [Python](https://github.com/apache/apisix-python-plugin-runner), [Go](https://apisix.apache.org/blog/2021/08/19/go-makes-apache-apisix-better/), etc.) through [Plugin Runner](https://apisix.apache.org/docs/apisix/external-plugin/). You will know at least one of these languages if you are a backend engineer. Using Plugin Runner, backend engineers can communicate through local RPC to develop APISIX plugins using the programming languages they are familiar with.
-
-The advantage of this is to reduce development costs and improve development efficiency. The disadvantage will be performance losses. So, is there a way to achieve the near-native performance of Lua using high-level languages that developers are familiar with?
-
-![Apache APISIX Plugin Runner](https://static.apiseven.com/2022/09/21/632ab5bddbb29.png)
-
-## Support WASM
+In APISIX, plugins are developed mainly through Lua, and [LuaJIT](https://apisix.apache.org/blog/2021/08/25/why-apache-apisix-chose-nginx-and-lua/) (a Just-In-Time Compiler for Lua) is used to ensure that the compiled code performance is good enough. 
```

Comment:
Update: removed in-depth details so that the audience can quickly jump to the core part of the article